device/common/pddf/plugins/sysstatutil.py

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env python
```

Comment:
Could we have comments of what each file in the plugins does ? It helps in easy understanding also it could be incremented with comment any new api's added

Reply:
sysstatusutil.py is not used as of now. Sure we will add the comments.